airbyte-integrations/connectors-performance/destination-harness/src/main/java/io/airbyte/integrations/destination_performance/PerformanceTest.java

```diff
@@ -149,7 +153,7 @@ void runTest() throws Exception {
         final AirbyteMessage airbyteMessage = new AirbyteMessage()
             .withType(Type.RECORD)
             .withRecord(new AirbyteRecordMessage()
-                .withStream(catalog.getStreams().get(0).getStream().getName())
+                .withStream(streamNames.get(random.nextInt(streamNames.size())))
```

Comment:
to confirm my understanding - this is the piece that mimics CDC by randomly returning streams?

Reply:
Yep, I confirmed this approach with Subodh, the caveat here is that it splits the dataset into the number of parallel streams